dev/fattest.simplicity/src/componenttest/topology/database/DerbyNetworkUtilities.java

```diff
@@ -16,43 +16,47 @@
 
 import org.apache.derby.drda.NetworkServerControl;
 
+import com.ibm.websphere.simplicity.log.Log;
+
 /**
  * Provides methods for starting and stopping Derby Network.
  * The server is checked a number of times to be sure it has started,
  * otherwise the exception returned from the ping is thrown.
  */
 public class DerbyNetworkUtilities {
+    private static final Class<?> c = DerbyNetworkUtilities.class;
 
     static final int NUMBER_OF_PINGS = 10;
     static final int SLEEP_BETWEEN_PINGS = 10000;
-    // TODO serverName and portNumber are hard coded, 
+    // TODO serverName and portNumber are hard coded,
     // but in the future should be taken from bootstrapping.properties if specified
     static final String serverName = "localhost";
     static final int portNumber = 1527;
 
     static public void startDerbyNetwork() throws UnknownHostException, Exception {
-        NetworkServerControl serverControl =
-                        new NetworkServerControl(InetAddress.getByName(serverName), portNumber);
+        final String m = "startDerbyNetwork";
+        NetworkServerControl serverControl = new NetworkServerControl(InetAddress.getByName(serverName), portNumber);
         serverControl.start(new PrintWriter(System.out));
         for (int i = 1; i <= NUMBER_OF_PINGS; i++) {
             try {
-                System.out.println("Attempt " + i + " to see if Derby Network server started");
+                Log.info(c, m, "Attempt " + i + " to see if Derby Network server started");
                 serverControl.ping();
                 break;
             } catch (Exception ex) {
                 if (i == NUMBER_OF_PINGS) {
-                    System.out.println("Derby Network server failed to start");
-                    ex.printStackTrace();
+                    Log.info(c, m, "Derby Network server failed to start");
+                    Log.error(c, m, ex);
                     throw ex;
+                } else {
+                    Log.warning(c, "Attempt " + i + " failed due to exception: " + System.lineSeparator() + ex.getMessage());
```

Comment:
Did you leave off the `m` argument for the 2nd parameter intentionally?

Reply:
Log.warning does not require the name of the method.  I believe it substitutes the word "warning"